index.js

```diff
@@ -9,9 +9,10 @@
   if (typeof define === 'function' && define.amd) {
     // AMD. Register as an anonymous module with d3 as a dependency.
     define([
-      'd3-collection',
```

Comment:
These changes to `d3-collection` and `d3-selection` should not be done without a more compelling reason.